test/jdk/javax/swing/JInternalFrame/6725409/bug6725409.java

```diff
@@ -147,9 +146,9 @@ private void checkResult() {
     private static void sync() {
         robot.waitForIdle();
     }
-    private static ExtendedRobot createRobot() {
+    private static Robot createRobot() {
         try {
-             ExtendedRobot robot = new ExtendedRobot();
+             Robot robot = new Robot();
              return robot;
          }catch(Exception ex) {
```

Comment:
```suggestion
         } catch(Exception ex) {
```
And pass the caught exception as the `cause` parameter to `Error` — `ex.printStackTrace()` becomes redundant.